components/app_bar/app_bar.scss

```diff
@@ -0,0 +1,61 @@
+.app-bar {
+    // Do not show App Bar if on mobile
+    @media (max-width: 768px) {
+        display: none;
+    }
+
+    padding-top: 40px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    border-left: 'solid 1px';
+    height: 100%;
+    width: 60px;
+    text-align: center;
+
+    .active-rhs-plugin {
+        border-left: 3px solid var(--button-bg) !important;
+    }
+
+    .fa, i {
+        font-size: 30px;
+        display: block;
+    }
+
+    .app-bar-icon {
+        display: inline-block;
+        color: rgba(var(--center-channel-color-rgb), 0.56);
+        fill: rgba(var(--center-channel-color-rgb), 0.56);
+        background: transparent;
+        border-left: 3px solid transparent;
```

Comment:
@michelengelen This is when the icon is in an inactive state. When it is in an active state, it will create this same border but with color `var(--button-bg)`.

I tried the padding you suggested instead, but this results in the icon shifting over to the right a bit when it becomes active. You can play around with this on this cloud server if you'd like: https://mattermost-webapp-pr-9273.test.mattermost.cloud